pkg/flag/report_flags.go

```diff
@@ -212,6 +227,7 @@ func (f *ReportFlagGroup) ToOptions(out io.Writer) (ReportOptions, error) {
 		ListAllPkgs:    listAllPkgs,
 		IgnoreFile:     getString(f.IgnoreFile),
 		ExitCode:       getInt(f.ExitCode),
+		ExitOnEosl:     getBool(f.ExitOnEosl),
```

Comment:
It is better to use the variable created above.

```suggestion
		ExitOnEosl:     exitOnEOSL,
```